docs/examples/imageset-config-helm.yaml

```diff
@@ -8,7 +12,4 @@ mirror:
         charts:
           - name: podinfo
             version: 5.0.0
-    local:
```

Comment:
Why remove the local example?

Reply:
I validated that each of these examples were functional. I'll add local back as commented so that it can still be functional while referencing the local example.